### ISSUE TITLE
README.md - Adding list of people should be followed in Web Accessibility

### ISSUE THREAD

**Issue by wendy-ha18** [codelikeagirlau/accessibility-resources#26]
Create a session to summary the list of People that has a lot of contributions to web accessibility should be followed.
